extras/lh_bench_23/parse_to_latex.py

```diff
@@ -26,6 +26,28 @@
 \end{table}
 """
 
+VFNS_LABELS = r"""
+    \multicolumn{1}{c|} {$xu_v$} &
+    \multicolumn{1}{c|} {$xd_v$} &
+    \multicolumn{1}{c|} {$xL_-$} &
+    \multicolumn{1}{c|} {$xL_+$} &
+    \multicolumn{1}{c|} {$xs_+$} &
+    \multicolumn{1}{c|} {$xc_+$} &
+    \multicolumn{1}{c|} {$xb_+$} &
+    \multicolumn{1}{c||}{$xg$} \\[0.5mm]
+    """
+
+FFNS_LABELS = r"""
+    \multicolumn{1}{c|} {$xu_v$} &
+    \multicolumn{1}{c|} {$xd_v$} &
+    \multicolumn{1}{c|} {$xL_-$} &
+    \multicolumn{1}{c|} {$xL_+$} &
+    \multicolumn{1}{c|} {$xs_v$} &
+    \multicolumn{1}{c|} {$xs_+$} &
+    \multicolumn{1}{c|} {$xc_+$} &
+    \multicolumn{1}{c||}{$xg$}
+    """
+
 
 def insert_haedrule(scheme, approx, caption):
```

Comment:
`insert_haedrule` -> `insert_headrule` :upside_down_face:

Reply:
done in 753475ab